Explicitly declare that older (legacy) versions of Traverse / Cinderscapes / Terraform are not supported

coderbot16 opened the pull request:
Users have reported cryptic crashes with these versions. An explicit "breaks" declaration seems a lot more intuitive.

Note: I haven't actually tested this change. It would be good to test this at least once before merging.

coderbot16 committed:
Explicitly declare that older (legacy) versions of Traverse / Cinderscapes / Terraform are not supported

Users have reported cryptic crashes with these versions. An explicit "breaks" declaration seems a lot more intuitive.

NeusFear reviewed (approved):
I also haven't tested but it seems like it's intuitive enough.

coderbot16 committed:
Fix "breaks" bounds using "<=" instead of "<"

Traverse 3.0.0 and Cinderscapes 1.2.3 are both compatible with the current version of Terrestria

Also add a breaks clause for old versions of Campanion. I didn't see any actual crashes with the latest 1.16.1 version, but I know that old versions of Campanion bundled an outdated version of Terraform.

Also, older versions of Campanion aren't compatible with 1.16.2+, so they shouldn't be used with Terrestria for 1.16.2+ anyways.

coderbot16 commented:
I've tested these changes and they seem to work. I've adjusted the bounds where appropriate.
